python/setup.py

```diff
@@ -2,11 +2,11 @@
 
 setup(
     name='Fledge',
-    python_requires='~=3.5',
-    version='0.1',
-    description='Fledge',
-    url='http://github.com/fledge/Fledge',
-    author='OSIsoft, LLC',
+    python_requires='>=3.6.9',
+    version='2.5.0',
+    description='Fledge, the open source platform for the Internet of Things',
+    url='https://github.com/fledge-iot/fledge',
+    author='Dianomic Systems Inc.',
```

Comment:
I'm not really sure, if We should replace or append?

Dianomic Systems, Inc.

Reply:
@MarkRiddoch Please suggest.

Reply:
We should probably append

Reply:
Also please confirm which one we should use for the following:
a) Dianomic Systems
b) Dianomic Systems Inc.
c) Dianomic Systems, Inc.